code/local.properties

```diff
@@ -4,5 +4,6 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-#Sun Mar 12 21:08:19 MDT 2023
+#Sat Mar 18 20:38:47 MDT 2023
+sdk.dir=/Users/mansoor/Library/Android/sdk
```

Comment:
Please delete your local sdk location before pushing to develop
Other than this everything looks good!  👍 
Great work!!

Reply:
will delete the sdk in a commit afterwards.